addons/spectator/initSettings.inc.sqf

```diff
@@ -42,7 +42,7 @@
     "LIST",
     [LSTRING(Sides), LSTRING(Sides_Description)],
     [LSTRING(DisplayName), LSTRING(Unconscious)],
-    [[0, 1, 2], [LSTRING(Friendly), LSTRING(Own), "str_all_voices"], 0],
+    [[0, 1, 3, 4, 2], [LSTRING(Friendly), LSTRING(Own), LSTRING(PlayerGroup), "str_player", "str_all_voices"], 0],
     1,
```

Comment:
Should `Own side` remain as the default here? Maybe we should change it?

Reply:
I have no opinion on this, can stay as i.s